homeworks/andrew.markovskiy_MarkoAndrew93/Games/styles.css

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
+
+a {
+  color: #EF4934;
+  text-decoration-line: underline;
+}
+
+
+p {
+  color: #221F1F;
+  line-height: 32px;
+  margin: 0;
+}
+
+
+.about {
+  width: 575px;
+  border: 2px #F00 solid;
```

Comment:
you can delete this border. We add this just to show what part of the page you need to do.